### ISSUE TITLE
Missing typed arguments in function NestedTreeRepository::__call()

### ISSUE THREAD

**Issue by ducho** [stof/StofDoctrineExtensionsBundle#473]
Missing typed arguments in function public function __call(string $method, array $args): mixed (edited). 

`SF64` `stof/doctrine-extensions-bundle 1.10.1` `doctrine/orm 3.0`

```Fatal error: Declaration of Gedmo\Tree\Entity\Repository\NestedTreeRepository::__call($method, $args) must be compatible with Doctrine\ORM\EntityRepository::__call(string $method, array $arguments): mixed in /var/www/symfony/vendor/gedmo/doctrine-extensions/src/Tree/Entity/Repository/NestedTreeRepository.php on line 67
Symfony\Component\ErrorHandler\Error\FatalError^ {#581
  #message: "Compile Error: Declaration of Gedmo\Tree\Entity\Repository\NestedTreeRepository::__call($method, $args) must be compatible with Doctrine\ORM\EntityRepository::__call(string $method, array $arguments): mixed"
  #code: 0
  #file: "./vendor/gedmo/doctrine-extensions/src/Tree/Entity/Repository/NestedTreeRepository.php"
  #line: 67
  -error: array:4 [
    "type" => 64
    "message" => "Declaration of Gedmo\Tree\Entity\Repository\NestedTreeRepository::__call($method, $args) must be compatible with Doctrine\ORM\EntityRepository::__call(string $method, array $arguments): mixed"
    "file" => "/var/www/symfony/vendor/gedmo/doctrine-extensions/src/Tree/Entity/Repository/NestedTreeRepository.php"
    "line" => 67
  ]
}
```

**Comment by tacman** [comment]
I'm getting this too, but I _think_ it's been fixed in either 1.11 or the main branch.  But I can't seem to install those

```
composer req stof/doctrine-extensions-bundle
./composer.json has been updated
Running composer update stof/doctrine-extensions-bundle
Loading composer repositories with package information
Updating dependencies                                 
Lock file operations: 2 installs, 0 updates, 0 removals
  - Locking gedmo/doctrine-extensions (v3.14.0)
  - Locking stof/doctrine-extensions-bundle (v1.10.1)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 2 installs, 0 updates, 0 removals
  - Installing gedmo/doctrine-extensions (v3.14.0): Extracting archive
  - Installing stof/doctrine-extensions-bundle (v1.10.1): Extracting archive
Generating optimized autoload files
```

I have minimum-stability at dev, but I think I need the latest of this bundle to avoid the error you're getting.

**Comment by stof** [comment]
`Gedmo\Tree\Entity\Repository\NestedTreeRepository` is not part of this bundle but of `gedmo/doctrine-extensions`, which is a separate package. you should report it to them instead.

Also note that 3.14.0 is not the latest version of those extensions. I doubt they will accept a bug report for an older version.

**Comment by tacman** [comment]
Reverting to doctrine/orm:^2 (possible now that https://github.com/jfsenechal/DoctrineBehaviors/pull/1 was merged) allows 3.15 to be installed, I'm adding this note in case anyone else runs into this.  It's _easy_ to run into, because 

    symfony new --webapp app

created a composer.json with doctrine/orm:^3
